cdap-ui/server/express.js

```diff
@@ -148,7 +148,7 @@ function makeApp (authAddress, cdapConfig) {
 
   // any other path, serve index.html
   app.all('*', [
-    httpIndexLogger,
+    //httpIndexLogger,
```

Comment:
I think this one should remain uncommented, so we get at least a little bit of positive logging.